From 361bf68e70308b63684e924f7f4aa27f1dfb755c Mon Sep 17 00:00:00 2001
From: ariel aviad <aaviad@habana.ai>
Date: Wed, 04 Sep 2024 18:06:18 +0300
Subject: [PATCH] [FSW-4085] sha: remove unnecessary parenthesis

remove parenthesis around the value
define SHA_SUM_LEN_ROM (32)

Change-Id: I59553627f93e623e5f6785dcfcf30e3de90207ac
Signed-off-by: ariel aviad <aaviad@habana.ai>
---

diff --git a/arch/gaudi2/common/sha.h b/arch/gaudi2/common/sha.h
index eb392e1..f69336c 100644
--- a/arch/gaudi2/common/sha.h
+++ b/arch/gaudi2/common/sha.h
@@ -7,7 +7,7 @@
 #include <preboot/fit_rsa.h>
 
 /* at ROM there are only 256 bits available in PSOC registers for keys hash */
-#define SHA_SUM_LEN_ROM			(32)
+#define SHA_SUM_LEN_ROM			32
 #define SHA_SUM_LEN			SHA384_SUM_LEN
 #define SHA_DER_LEN			SHA384_DER_LEN
 #define SHA_DER_PRE			sha384_der_prefix
